Release 0.10.0 (#237)
samply/beam: CHANGELOG.md

# Samply.Beam 0.9.2 - 2025-05-25

## Minor changes

> * Improved error message when the beam certificate expired.

Reviewer: ```suggestion
* Improved Beam Proxy's error message when its beam certificate expired.
```

> * Improved Readme to mention companion tools.

Reviewer: ```suggestion
* Mention Beam companion tools in Readme.
```

## Bugfixes

> * Fixed a bug where the proxy would crash when its beam certificate expired even if it had been resigned.

Reviewer: ```suggestion
* Fixed a bug where the proxy would crash when its beam certificate expired even if it had been re-signed.
```
